docs/source/developer/fields.md

```diff
@@ -306,3 +307,61 @@ Delete:
     }
 }
 ```
+
+
+## Bucket dict field
+
+```python
+from zope.interface import Interface
+from guillotina.fields import BucketDictField
+from guillotina import schema
+
+class IMySchema(Interface):
+    values = BucketDictField(
+        value_type=schema.Text(),
```

Comment:
`key_type` is also usable with `BucketDictField` no?

Reply:
I think I need to consider this. We need to force a key type that is consistently comparable when doing the bucket splitting...

So str, int, float, datetime would work.

Should we have checks for this or buyer beware or...?